api/v1beta1/ovs_types.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	"github.com/openstack-k8s-operators/lib-common/modules/common/condition"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// OVSSpec defines the desired state of OVS
+type OVSSpec struct {
+	// +kubebuilder:validation:Optional
+	ExternalIDS OVSExternalIDs `json:"external-ids"`
+
+	OvsContainerImage string `json:"ovsContainerImage"`
+
+	OvnContainerImage string `json:"ovnContainerImage"`
+
+	// +kubebuilder:validation:Optional
```

Comment:
just a nit, should those have comments which are shown in the spec to the enduser as an explanation of their use?